DATA_CONTRACT.md

```
@@ -107,3 +107,21 @@ These files contain system logic, scripts, templates, and instructions that impr
 **If a file is in the User Layer, no update process may read, modify, or delete it.**
 
 **If a file is in the System Layer, it can be safely replaced with the latest version from the upstream repo.**
+
+## Custom Data Directory
+
+By default, all User Layer files reside inside the project repository root. You can configure a separate, external directory for your personal data.
+
+### Resolution Precedence:
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)